Sources/DeepLearning/Layer.swift

```diff
@@ -1297,12 +1297,12 @@ public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
 
     @noDerivative var stateShape: TensorShape
 
-		@differentiable
-		public var zeroState: Tensor<Scalar> {
-				get {
-						return Tensor(zeros: stateShape)
-				}
+	@differentiable
+	public var zeroState: Tensor<Scalar> {
+		get {
+			return Tensor(zeros: stateShape)
```

Comment:
I think you are using tabs instead of spaces.

Reply:
Yep. Fixed.